source/releasenotes/2025-05-07-terminus-4-0-0.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Terminus 4.0.0 release now available"
+published_date: "2025-05-07"
+categories: [tools-apis]
+---
+
+Terminus [4.0.0](https://github.com/pantheon-systems/terminus/releases/tag/4.0.0) is now available.
+The next generation of Terminus adds support for PHP 8.4 and removes support for PHP versions 8.1 and below.
```

Comment:
Is it worth mentioning how this relates to Pantheon sites running PHP 7.x and below?

Reply:
Since terminus runs on local machines, the version of PHP that's running on Pantheon application servers seems less relevant (and probably confuses things)

Reply:
I think we should explain that